SchedulerJobs/SchedulerJobs.Services/CloseConferenceService.cs

```diff
@@ -31,6 +34,29 @@ public async Task<int> CloseConferencesAsync()
 
             return conferenceCount;
         }
+
+        public async Task<int> DeleteAudiorecordingApplicationsAsync()
+        {
+            var conferences = await _videoApiService.GetExpiredAudiorecordingConferences();
+            var conferencesCount = 0;
+            if (conferences != null && conferences.Any())
```

Comment:
.Any() will enumerate again, just use .Count > 0

Reply:
In Count() method code will traverse all the list and get total number of objects in list while in Any() will return after examining first element in the sequence. So in list where we have many object it will be significant execution time if we use count().

Reply:
Thats true, but in this case you can use .Count > 0
NOT .Count()
Like you have done 2 line below

Reply:
Any() for me is a good option when all that we want to know is whether the list is empty or not.